aten/src/ATen/FunctionalTensorWrapper.h

```diff
@@ -124,7 +123,9 @@ TORCH_API bool isFunctionalTensor(const c10::List<c10::optional<Tensor>>& t_list
 TORCH_API bool isFunctionalTensor(const c10::ArrayRef<Tensor> t_list);
 
 TORCH_API Tensor to_functional_tensor(const Tensor& tensor);
+TORCH_API c10::optional<Tensor> to_functional_tensor(const c10::optional<Tensor>& tensor);
```

Comment:
How does C++ pick which overload to use if you pass in to_functional_tensor(tensor)? (this doesn't really matter because the code compiles but I'm curious)